web/thesauruses/lua/5/control_structures.json

```diff
@@ -0,0 +1,120 @@
+{
+  "meta": {
+    "language": "lua",
+    "language_name": "Human-Readable Language Name",
```

Comment:
You meant to update this from "Human-Readable Language Name", right? :)

Reply:
Huh? I'm pretty sure i edited this, ... I wonder which git command I used to undo this.